benefits/templates/admin/agency-dashboard-index.html

```diff
@@ -1,51 +1,10 @@
-{% extends "admin/base.html" %}
+{% extends "in_person/base.html" %}
```

Comment:
Going along with the comment above, this would become

```suggestion
{% extends "admin/base.html" %}
```